src/App.tsx

```diff
@@ -8,27 +8,27 @@ import Column from "./components/Column";
 import Project from "./components/Project";
 
 const GlobalStyle = createGlobalStyle`
-  font-family: Open Sans, Montserrat, Roboto
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Open Sans, Montserrat, Roboto;
+}
```

Comment:
We had a little weird 8px margin in the outermost place coming from <body />. Updated global style to remove such space